### PR TITLE
block multimedia submissions

### PR DESCRIPTION
**Pull request by NoahCarnahan**

### CODE REVIEW

File: corehq/ex-submodules/couchforms/getters.py [hunk 1]
```diff
@@ -30,6 +32,11 @@ def get_instance_and_attachment(request):
     if request.META['CONTENT_TYPE'].startswith('multipart/form-data'):
         # ODK submission; of the form
         # $ curl --form 'xml_submission_file=@form.xml' $URL
+
+        if request.POST.keys():
+            notify_error("Received a submission with POST.keys()")
+            raise Exception("Received a submission with POST.keys()")
```
Comment (czue) [inline corehq/ex-submodules/couchforms/getters.py]: Thoughts on raising a more specific error here, and then trapping it higher up the chain and responding with a `BadRequest` or something less loud?

Comment (NoahCarnahan) [inline corehq/ex-submodules/couchforms/getters.py]: Yeah that sounds better

**Comment by czue** [comment]
no blockers, but am thinking it may make sense to wait on tests either way since this is such a path-critical piece of code

**Comment by NoahCarnahan** [comment]
@czue Just updated to respond with a 400 instead of 500

**Comment by NoahCarnahan** [comment]
(Killed first build)

**Comment by czue** [comment]
lgtm.

**Comment by gcapalbo** [comment]
did the tests ever run on this? i think the build on master is failing with this change:
```
======================================================================
FAIL: testSubmitBadAttachmentType (corehq.apps.receiverwrapper.tests.test_submit_errors.SubmissionErrorTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/corehq/util/test_utils.py", line 227, in inner
    return helper(*args, **kwargs)
  File "/mnt/corehq/util/test_utils.py", line 200, in __call__
    call_with_settings(fn_with_pre_and_post, run_config.settings, args, kwargs)
  File "/mnt/corehq/util/test_utils.py", line 215, in call_with_settings
    fn(*args, **kwargs)
  File "/mnt/corehq/util/test_utils.py", line 196, in fn_with_pre_and_post
    self.fn(*args, **kwargs)
  File "/mnt/corehq/apps/receiverwrapper/tests/test_submit_errors.py", line 46, in testSubmitBadAttachmentType
    self.assertIn("xml_submission_file", res.content)
AssertionError: 'xml_submission_file' not found in 'Received a submission with POST.keys()'```

**Comment by NoahCarnahan** [comment]
@gcapalbo Sorry about that. Build fixed here: https://github.com/dimagi/commcare-hq/pull/10595/

**Pull request merged**